### ISSUE TITLE
Request: Pixels

### ISSUE THREAD

**Issue by Kythe42** [simple-icons/simple-icons#11818]
### Brand Name

Pixels

### Website

https://pixels.com/

### Popularity Metric

The similarweb rank is 39,059.

https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/3m?webSource=Total&key=pixels.com

### Official Resources for Icon and Color

PNG: https://pixels.com/assets/images/logos/logoPixelsBlack2020.png

**Comment by uncenter** [comment]
https://pixels.com/termsofuse.html:

> Any name, logo, trademark, service mark, patent, design, copyright, domain name or other intellectual property appearing on this Website is owned or licensed by Pixels or its parents, affiliates, or subsidiaries and may not be used by You without the prior written consent of Pixels or the appropriate owner. Your use of this Website does not grant You any right, title, interest or license to any such intellectual property appearing on this Website. Any unauthorized use of the content of this Website may subject You to civil or criminal penalties.

We will need to [reach out & request permission](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission) - @Kythe42 would you be willing to do so?

**Comment by Kythe42** [comment]
Confirming that I have submitted a permission request through the contact form on the company website. As Pixels.com is the sister site of FineArtAmerica.com I have jointly requested permission for requests #11817 and #11818 on the same contact form.

**Comment by Kythe42** [comment]
Frustratingly, there has been no response after a week, so I've sent a followup email.

**Comment by Kythe42** [comment]
I did get a reply from Fine Art America / Pixels after my follow up email. Unfortunately, the agent seemed to completely misunderstand what I was asking despite using the form letter, and I was given instructions on how to embed a shopping cart in my website. I replied trying my best to explain what this project does and how it could benefit both their companies and their users and asked if creating icons from their logos would be allowed. I'll paste the next reply I received.

> Hello Rowan,
> 
> Thank you for clarifying this. We don't have this feature at the moment, however, we will take this as a suggestion, which I am passing over to the Production Team now. They may not get back to you personally, but please know that they will see it when reviewing all the suggestions.
> 
> 
> Let me know if you need further assistance.
> 
> Regards,
> Red

I still feel like they don't fully understand what I was asking permission for and the reply seems to indicate that he's looking into getting the company to make the icons for Fine Art America / Pixels. This is why I was hesitant about reaching out myself to request permission as I don't feel qualified to properly represent this project, and I'm at a loss as to how to proceed at this point.

If there's any other reply you think I should give to the agent, please let me know what to say, or let me know if I should just leave it. Thanks.

**Comment by Kythe42** [comment]
No, I have not. As I previously indicated it didn't feel like they understood what I was asking permission for. I asked for suggestions on other things I could say to explain it better and I was given no feedback. I think someone else should give this a shot that's better at communicating and understands the nuances of this project better than I do.

**Comment by PeterShaggyNoble** [comment]
Unfortunately, we have to interpret lack of explicit permission as a denial of permission. If someone else is able to gain permission in the future then we can revisit this issue.
